defaults/main.yml

```diff
@@ -23,6 +23,9 @@ galaxy_manage_gravity: "{{ false if __galaxy_major_version is version('22.05', '
 galaxy_manage_systemd: no # For Galaxy
 galaxy_manage_systemd_reports: no # For Reports
 galaxy_manage_cleanup: no
+galaxy_manage_themes: no
+galaxy_manage_static: no
```

Comment:
just catching up, this is semantically different than `galaxy_manage_static_setup`?  They sound so similar, I'm wondering if they shouldn't be the same variable?

Reply:
Maybe the naming is a bit bad here. The variable enables the creation of static subdomain directories

Reply:
I changed it, maybe this is better @hexylena ?